classes/class-wpcom-liveblog-entry.php

```diff
@@ -120,7 +136,7 @@ public function for_json() {
 			'css_classes' => $css_classes,
 			'timestamp'   => $this->get_timestamp(),
 			'authors'     => self::get_authors( $entry_id ),
-			'entry_time'  => get_comment_date( 'U', $entry_id ),
+			'entry_time'  => $this->get_comment_date_gmt( 'U', $entry_id ),
```

Comment:
Should `get_fields_for_render()` be updated to use this helper, too?

Reply:
Just looking into this, I don't think this function `get_fields_for_render()` is used anymore as the rendering is now done with react but it might be worth leaving this function in if rendering in php is required again in the future, the `for_json` function is used for the react render. I've also added phpcs fixes in.

Reply:
I was wondering why markup is sent over the wire and not used. It would be great to remove that now-dead code.